src/optimizer-page/data/api.test.js

```diff
@@ -0,0 +1,47 @@
+import MockAdapter from 'axios-mock-adapter';
+import { initializeMockApp, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+import { getExportStatus, postExportCourseApiUrl, startCourseExporting } from './api';
+
+let axiosMock;
+const courseId = 'course-123';
+
+describe('API Functions', () => {
+  beforeEach(() => {
+    initializeMockApp({
+      authenticatedUser: {
+        userId: 3,
+        username: 'abc123',
+        administrator: true,
+        roles: [],
+      },
+    });
+    axiosMock = new MockAdapter(getAuthenticatedHttpClient());
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
Instead of all this boilerplate, you can just use

```tsx
const { axiosMock } = initializeMocks();
```

as the first line of each test case. You can then remove the `beforeEach` and `afterEach` you have here, because `initializeMocks` takes care of that all for you, including `jest.clearAllMocks();`.

See

https://github.com/openedx/frontend-app-authoring/blob/f86c609ff147348e15c1a51bbd64f79b4693bdcc/src/testUtils.tsx#L152-L162